tiledb/api/c_api/subarray/subarray_api_experimental.h

```diff
@@ -66,6 +66,40 @@ TILEDB_EXPORT capi_return_t tiledb_subarray_add_point_ranges(
     const void* start,
     uint64_t count) TILEDB_NOEXCEPT;
 
+/**
+ * Adds variable-sized point ranges to the given dimension index of the
+ * subarray. Effectively `add_range(x_i, x_i)` for all points in the
+ * target array, but set in bulk to amortize expensive steps.
+ *
+ * **Example:**
+ *
+ * @code{.c}
+ * const char* buffer_val = "TileDBtest";
```

Comment:
I'd suggest to use a buffer value of "aabcccddee" and explain that the actual ranges to be added will be [aa,aa], [b,b] [ccc, ccc] [dd, dd] and [ee, ee]